UserAgent/Views/QuerySuggestions/SuggestionsView.swift

```diff
@@ -0,0 +1,176 @@
+//
+// Copyright (c) 2017-2019 Cliqz GmbH. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+
+import UIKit
+
+private extension String {
+    func trim() -> String {
+        let newString = self.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
+        return newString
+    }
+}
+
+class SuggestionsView: UIView {
+    //MARK:- instance variables
+    var onSuggestionTapped: ((_ suggestion: String) -> Void)?
+
+    //MARK:- Constants
+    private let boldFontAttributes = [
+        NSAttributedString.Key.font: UIFont.boldSystemFont(ofSize: 17),
+        NSAttributedString.Key.foregroundColor: UIColor.white,
+    ]
+    private let normalFontAttributes = [
+        NSAttributedString.Key.font: UIFont.systemFont(ofSize: 16),
+        NSAttributedString.Key.foregroundColor: UIColor.white,
+    ]
+    private let separatorBgColor = UIColor(rgb: 0xC7CBD3)
+    private let margin: CGFloat = 10
+
+    //MARK:- private variables
+    private var currentSuggestions: [String] = []
+
+    private var currentQuery: String = "" {
+        didSet {
+            if currentQuery.isEmpty {
+                currentSuggestions.removeAll()
+                clearSuggestions()
+            }
+        }
+    }
+
+    //Mark:- public methods
```

Comment:
Small nitpick but this should read 

```suggestion
    // MARK: - Public API
```

the letter case and spacing is actually important for `// MARK: -` comments